src/pytorch_lightning/strategies/ddp_spawn.py

```diff
@@ -68,6 +68,7 @@ def __init__(
         ddp_comm_hook: Optional[callable] = None,
         ddp_comm_wrapper: Optional[callable] = None,
         process_group_backend: Optional[str] = None,
+        start_method: str = "spawn",
```

Comment:
with another start method, this isn't really ddp_*spawn* anymore, is it? :D

Reply:
Yes, but I would not rename the DDPSpawn strategy class yet. Long-term, I believe it will eventually be merged together with the DDP strategy. The names in the registry remain consistent though, so we will have "ddp_spawn" and "ddp_fork" as string names.

What we should do is rename the SpawnLauncher. It's a protected API so should be no problem there.